packages/Button/Button.jsx

```diff
@@ -94,6 +199,7 @@ Button.propTypes = {
 Button.defaultProps = {
   type: 'button',
   variant: 'primary',
+  rank: 'common',
```

Comment:
as per [this doc](https://docs.google.com/document/d/19sZaR43PT6poWr0ykjgcVio_ECdVaB7TrlNCZ78O-uk/edit#), buttons by default are supposed to be of type `rank="main"`, why are we passing `rank="common"` as default prop?

Reply:
default is common, discussion on slack to confirm